docker/generate_build_matrix.py

```diff
@@ -92,10 +92,9 @@ def _get_cuda_override(cuda_version: str):
                                     'brand=quadro,driver>=515,driver<516 brand=titan,driver>=470,driver<471 '
                                     'brand=titan,driver>=515,driver<516 brand=titanrtx,driver>=470,driver<471 '
                                     'brand=titanrtx,driver>=515,driver<516')
-
         return cuda_118_override_string
-
-    return ''
+    else:
+        return ''
```

Comment:
```suggestion
    return ''
```
nit, no need for else because it's a return switch